static/js/addons/cl.utils.js

```diff
@@ -49,8 +49,12 @@ var Cl = window.Cl || {};
         },
 
         // INFO: simple mobile and tablet filter that can be extended
-        mobile: function () { return (navigator.userAgent.match(/iPhone/i) || navigator.userAgent.match(/Android/i)); },
-        tablet: function () { return (navigator.userAgent.match(/iPad/i)); }
+        mobile: function () {
+            return (navigator.userAgent.match(/iPhone/i) || navigator.userAgent.match(/Android/i));
```

Comment:
Yes this is normally managed using responsive techniques such as `if ($(window).width() > 1024px)` rather than device detection. There is also a cool library available from @vxsx https://github.com/vxsx/bootstrap-breakpoints